repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -25,13 +27,15 @@
         "_virtual_versions": {
             "8": "8.10",
             "9": "9.6",
+            "9.7": ["10.1"],
```

Comment:
this is also invalid. check virtual versions around. virtual version != upgrade path. It's virtual adding of a minor version to centos major version.